tensorflow/lite/examples/label_image/README.md

```diff
@@ -78,4 +78,86 @@ average time: 68.12 ms
 0.00608029: 543 543:drumstick
 ```
 
+## Run the sample on an Android device
+Prepare data on devices, e.g.,
+
+```
+adb push bazel-bin/tensorflow/lite/examples/label_image/label_image  /data/local/tmp
+adb push /tmp/mobilenet_v1_1.0_224.tflite  /data/local/tmp
+adb push tensorflow/lite/examples/label_image/testdata/grace_hopper.bmp  /data/local/tmp
+adb push /tmp/labels.txt /data/local/tmp
+```
+
+Run it,
+```
+adb shell "/data/local/tmp/label_image \
+-m /data/local/tmp/mobilenet_v1_1.0_224.tflite \
+-i /data/local/tmp/grace_hopper.bmp \
+-l /data/local/tmp/labels.txt"
+```
+then you should see something like the followings:
+```
+Loaded model /data/local/tmp/mobilenet_v1_1.0_224.tflite
+resolved reporter
+INFO: Initialized TensorFlow Lite runtime.
+invoked
+average time: 25.03 ms
+0.907071: 653 military uniform
+0.0372416: 907 Windsor tie
+0.00733753: 466 bulletproof vest
+0.00592852: 458 bow tie
+0.00414091: 514 cornet
+```
+
+Run the model with NNAPI delegate (`-a 1`),
+```
+adb shell "/data/local/tmp/label_image \
+-m /data/local/tmp/mobilenet_v1_1.0_224.tflite \
+-i /data/local/tmp/grace_hopper.bmp \
+-l /data/local/tmp/labels.txt -a 1 -f 1"
+```
+then you should see something like the followings:
+```
+Loaded model /data/local/tmp/mobilenet_v1_1.0_224.tflite
+resolved reporter
+INFO: Initialized TensorFlow Lite runtime.
+INFO: Created TensorFlow Lite delegate for NNAPI.
+Applied NNAPI delegate.invoked
+average time: 10.348 ms
+0.905401: 653 military uniform
+0.0379589: 907 Windsor tie
+0.00735866: 466 bulletproof vest
+0.00605307: 458 bow tie
+0.00422573: 514 cornet
+```
+
+To run a model with the Hexagon Delegate, assuming we have
+followed the
+[Hexagon Delegate doc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/performance/hexagon_delegate.md)
+and installed Hexagon libraries in `/data/local/tmp`. Run it
+```
+adb shell "export LD_LIBRARY_PATH=/data/local/tmp/; \
```

Comment:
We can get rid of the "export LD_LIBRARY_PATH=/data/local/tmp/" by adding rpath=/data/local/tmp/ to the linkopts in the build rule.
Like here
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/benchmark/BUILD#L32